### ISSUE TITLE
LittleFS mount fail on new device

### ISSUE THREAD

**Issue by cad435** [SasaKaranovic/winfidel-sensor#2]
Hello!

After building the devices (PnP as per BOM, used JLCPCBA) i get the following error:

```Starting...
*wm:AutoConnect
*wm:Connecting to SAVED AP: ***
*wm:connectTimeout not set, ESP waitForConnectResult... 
*wm:AutoConnect: SUCCESS 
*wm:STA IP Address: 192.168.178.72
Connected.
Starting LittleFS...E (11254) esp_littlefs: ./components/esp_littlefs/src/littlefs/lfs.c:1367:error: Corrupted dir pair at {0x0, 0x1}

E (11255) esp_littlefs: mount failed,  (-84)
E (11258) esp_littlefs: Failed to initialize LittleFS
[ 11148][E][LittleFS.cpp:98] begin(): Mounting LittleFS failed! Error: -1
An Error has occurred while mounting LittleFS
```

Any thoughts what went wrong?

**Comment by cad435** [comment]
Just realized I forgot to build and upload the filesystem contents...

PEBCAK error I suppose :)

**Comment by SasaKaranovic** [comment]
Been there done that. :)
Glad you figured it out. Let me know how it works.

Would you mind sharing how much did the PCBA run cost?
I know there are others that are interested in ordering these. I received couple of emails asking how much would it cost to make 1/5/10 of these through one of the online PCB services.

**Comment by cad435** [comment]
Sure!

5 pcs cost me 36.84€ shipped.

Mind you, the USB-C connector, ADC, LDO, the tactile swithces and the 8pf capacitors I soldered myself. I have those laying around and did not want to spend the additional cost on those.
The problem is, those are "non standard" components for JLC so they take 3$ setup fee per component, which would increase the cost by 15$. 
I was willing to take that for the ESP and the Antenna as I had not laying those around.



if you go all in I'd suppose you'll end up on about 50-60€ shipped for 5 pcs and about 90-100€ for 10pcs.


Edit:
This is the CPL file I generated with Altium and used - I uploaded the same File to BOM and CPL on JLC's website
[JLC_PCBA_CPL-ESP_Infidel.xlsx](https://github.com/SasaKaranovic/winfidel-sensor/files/15287484/JLC_PCBA_CPL-ESP_Infidel.xlsx)

You might want to complete the last column with LCSC's C-Numbers (PN's) for a "klick and upload" esperience :)
